haproxy/datadog_checks/haproxy/version_utils.py

```diff
@@ -0,0 +1,25 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+import re
+
+from datadog_checks.base.log import get_check_logger
+
+VERSION_PATTERN = re.compile(r"(?:HAProxy|hapee-lb) version ([^,]+)")
+
+
+def get_metadata_from_http(raw_version):
+    if raw_version == "":
+        get_check_logger().debug("unable to find HAProxy version info")
```

Comment:
`get_check_logger()` is mainly intended to be instantiated once per check. Example via a config object: https://github.com/DataDog/integrations-core/blob/c42a957be969b6cf5d2bc239bb04ab292664378f/mysql/datadog_checks/mysql/config.py#L12

Calling it for every check run might have a cost (frame lookup).

I think logging in `_set_metadata` might be more suitable than util methods.

Reply:
ok